testing/dart/fragment_shader_test.dart

```diff
@@ -332,34 +332,6 @@ void main() async {
     shader.dispose();
   });
 
-  // This test can't rely on actual pixels rendered since it needs to run on a
```

Comment:
Is the test on line 275 sufficient? Does that test need to have a sampler or two thrown in?

Reply:
wouldn't apply either way, its not running on impeller or metal?